satpy/node.py

```diff
@@ -423,12 +423,10 @@ def _find_dependencies(self, dataset_key, **dfilter):
         except KeyError:
             # exact dataset isn't loaded, let's load it below
             pass
-
```

Comment:
Why were these lines removed? Automatic by some auto-correcting tool? Please add the blank lines back in unless they were style errors.